app/customer.py

```diff
@@ -0,0 +1,49 @@
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            car: dict
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = Car(*car.values())
+        print(f"{self.name} has {self.money} dollars")
+
+    def trip_cost(
+            self,
+            shop: Shop,
+            fuel_price: float
+    ) -> float:
+        products_cost = shop.products_cost(self.product_cart).values()
+        fuel_cons = self.car.total_consumption(self.location, shop.location)
```

Comment:
The `total_consumption` method in the `Car` class calculates the round trip fuel consumption. Ensure that this aligns with your requirement to calculate the cost for a round trip, as per the task description.